frontend/src/main/components/Jobs/jobMilkCows.js

```diff
@@ -0,0 +1,8 @@
+function jobMilkCows() {
```

Comment:
I think that the file name and function name should be JobMilkCows as referenced in the following form

Reply:
But looks good !

Reply:
oh sorry that was my fault, i suggested that. We can change that then

Reply:
Resovled.